install_config/install/advanced_install.adoc

```diff
@@ -2489,10 +2489,22 @@ specific components as needed. By breaking up the roles and playbooks, there is
 better targeting of ad hoc administration tasks. This results in an increased
 level of control during installations and results in time savings.
 
-[NOTE]
-====
 The playbooks and their ordering are detailed below in
 xref:running-the-advanced-installation-individual-components[Running Individual Component Playbooks].
+
+
+[NOTE]
+====
+After running the installation, due to a known issue, the following directories might get created:
```

Comment:
I'd clarify this.
`If NFS volumes are provisioned for any component, all of the following directories will be created whether their components are being deployed to NFS volumes or not.`

Reply:
@sdodson Thank you!